sdks/java/testing/test-utils/src/test/java/org/apache/beam/sdk/testutils/jvmverification/JvmVerification.java

```diff
@@ -39,6 +40,7 @@ public class JvmVerification {
     versionMapping.put("0034", v1_8);
     versionMapping.put("0037", v11);
     versionMapping.put("003d", v17);
+    versionMapping.put("0041", v21);
```

Comment:
Nonblocking since it seems to be consistently hex `002c + version`, but where are these coming from?

Reply:
these are java bytecode versions. Yes from Java8 its been 44 (2c in hex) + version though no guarantee this trand continues. ref: https://javaalmanac.io/bytecode/versions/